recipes/browsermob.rb

```diff
@@ -39,8 +39,6 @@
 # Create dedicated user
 user node['aet']['browsermob']['user'] do
   group node['aet']['browsermob']['group']
-  manage_home true
```

Comment:
What is the reason for deleting this line? Recently, when we tried to use this cookbook on non-vagrant environment without the `_develop` recipe (which creates `developer` user/group), we needed to *add* following lines to the `karaf user` creation in `karaf` recipe - otherwise Karaf was not started successfully.
```
  manage_home true
  home node['aet']['karaf']['root_dir']
```
(you can see full updated file here: https://github.com/PhilBug/aet-cookbook/blob/c7df76f64fd2d40b26f4dd0502b042ecf9129455/recipes/karaf.rb)

Reply:
Thank you @tkaik ! This is very good observation !
After investigation it looks that the issue is related to Karaf installing features.
See:

* [Procedure for locating artifacts](https://access.redhat.com/documentation/en-us/red_hat_jboss_fuse/6.2/html/deploying_into_the_container/esbdeploylocateruntime#topic-36484)
* [Local repository](https://access.redhat.com/documentation/en-us/red_hat_jboss_fuse/6.2/html/deploying_into_the_container/ESBDeployLocateRunTime#Locate-RunTime-LocalRepo)

The mentioned property is **org.ops4j.pax.url.mvn.localRepository** and should be configured in `.../<karaf folder>/etc/org.ops4j.pax.url.mvn.cfg`

Karaf has a description for it:

> Path to the local Maven repository which is used to avoid downloading
> artifacts when they already exist locally.
> The value of this property will be extracted from the settings.xml file
> above, or defaulted to:
>     System.getProperty( "user.home" ) + "/.m2/repository"
> leaving this option commented makes the system dependent on external
> configuration, which is not always desired
> "localRepository" is the target location for artifacts downloaded from
> "remote repositories"

By default the property is NOT configured - that is why adding home for Karaf user solved the issue. In my opinion Karaf should be run under *system* user and home folder is not needed here. In the same way home folder is not needed for Browsermob system user.

Because the issue you described exists on **master** branch i will create another pull request with fix.

Reply:
I have also updated this pull request to contain fix for Karaf home folder issue.